src/js/services/app-service.js

```diff
@@ -2,12 +2,16 @@ import { store } from 'stores/app-store';
 import * as AppStateActions from 'actions/app-state-actions';
 import { observable } from 'utils/events';
 
+import { preventPageScroll, allowPageScroll } from 'utils/dom';
+
 
 export function openWidget() {
     let {embedded} = store.getState().appState;
     if (!embedded) {
         store.dispatch(AppStateActions.openWidget());
         observable.trigger('widget:opened');
+
+        preventPageScroll();
```

Comment:
I would have the is apple device here, out of context if hard to understand why you would prevent page scroll